src/collections/_documentation/platforms/python/asgi.md

```diff
@@ -0,0 +1,34 @@
+---
+title: ASGI
+sidebar_order: 5
+---
+
+{% version_added 0.10.2 %}
+
+<!-- WIZARD -->
+
+The ASGI middleware can be used to instrument any [ASGI](https://asgi.readthedocs.io/en/latest/)-compatible web framework to attach request data for your events.
+
+This can be used to instrument for example [Starlette](https://www.starlette.io/middleware/) or [Django Channels 2.0](https://channels.readthedocs.io/en/latest/).
```

Comment:
```suggestion
This can be used to instrument, for example [Starlette](https://www.starlette.io/middleware/) or [Django Channels 2.0](https://channels.readthedocs.io/en/latest/).
```